app/questionnaire/router.py

```diff
@@ -350,7 +350,15 @@ def _get_return_to_for_calculated_summary(
             # remove first item and return the remaining ones
             # Type ignore: return_location.return_to and return_location.return_to_answer_id will always be populated at this point
```

Comment:
This type ignore string is no longer correct?

Reply:
Also, a note to self on being careful with type ignores. It was an issue highlighted by mypy that we chose to ignore and got wrong.

Reply:
Yeah, it is guaranteed, this code only executes when higher up this `if` statement is valid:
```python
if return_location.return_to.startswith("calculated-summary")..
```